### PR TITLE
Enemy improvements

### LINKED ISSUE #87: Enemy improvements
Playtesting revealed that:
- Enemies aren't being distracted
- Enemies can pass through walls in the freeform tutorial

### PR DESCRIPTION
**Pull request by StevieGoodman**

### CODE REVIEW

File: Assets/Scripts/Characters/EnemyController.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ private void Update()
             enemyStateCanvas.transform.rotation = _camera.transform.rotation;
         }
 
-        public async void FixedUpdate()
```
Comment (Vesper-Works) [inline Assets/Scripts/Characters/EnemyController.cs:47]: lol

Comment (Vesper-Works) [inline Assets/Scripts/Characters/EnemyController.cs:47]: lol

**Comment by Vesper-Works** [comment]
I'm gonna move the player back to spawn

**Comment by StevieGoodman** [comment]
Should already be at spawn

**Comment by Vesper-Works** [comment]
The player wasn't, but now is

**Review by Vesper-Works — APPROVED** [review]

**Comment by Vesper-Works** [comment]
Now you've gotta approve since I pushed last

**Comment by StevieGoodman** [comment]
Can't, I'm the branch owner lol. just gonna override

**Pull request merged**